lib/tasks/import.rake

```diff
@@ -13,12 +15,15 @@ namespace :import do
       organisation_content_id, args.document_type, document_subtypes
     )
 
-    documents_to_import = WhitehallMigration::DocumentImport.where(whitehall_migration_id: whitehall_migration.id).count
+    documents_to_import = WhitehallMigration::DocumentImport.where(whitehall_migration_id: whitehall_migration).count
```

Comment:
s/whitehall_migration_id/whitehall_migration/

Reply:
Updated.  Also tested running the rake test manually to ensure a useable URL is printed.